src/utils/echis.js

```diff
@@ -0,0 +1,61 @@
+const axios = require("axios");
+
+function genRequestConfig(conf) {
+    return {
+        baseURL: conf.instance,
+        auth: {
+            username: conf.user,
+            password: conf.password,
+        },
+    }
+}
```

Comment:
Let us stick to the arrow notation for consistency.

Reply:
done